README.md

```diff
@@ -1,2 +1,64 @@
-# handbook-of-best-practices
-Software Quality Assurance Assignment
+### 5. Limited use of globals
+
+---
+
+Code is generally clearer and easier to maintain when it does not use globals. Global variables can be read or modified by any part of the program, making it difficult to identify the reason for every time it is used. Because global variables can be get or set by any part of the program, any rules regarding its use can be easily broken. Also, because globals are available everywhere, you might unknowingly end up using it when you think you are using a local variable (or vice versa).
+
+&nbsp;
+
+### 6. Code Comments and Proper Documentation
+
+---
+
+It is good to comment while writing code as it helps other developers to understand your code. It is good practice to start every method or function with a comment specifying what the method or function does, about its parameters and return values. Also the role of each class and file, their contents and any steps of complex code should be commented. For example, 
+
+```java
+/* this function will used to divide two variables */
+
+int Divide()
+{
+
+//logic of the function
+
+}
+```
+
+&nbsp;
+
+### 7. Proper and Consistent Scheme for Naming
+
+---
+
+The main naming conventions are
+
+camelCase: This is where the first letter of each word is capatalised, except for the first word.
+
+under_score: This is naming by using an underscore between the words.
+
```

Comment:
Could you add a colon and bullet points for each of the examples.

Reply:
done